it/controllers/AppealsControllerISpec.scala

```diff
@@ -179,6 +179,13 @@ class AppealsControllerISpec extends IntegrationSpecCommonBase with ETMPWiremock
       result.body shouldBe appealJson.toString()
     }
 
+    "call ETMP and compare the penalty ID provided and the penalty ID in the payload with multiple Penalty Period - return OK if there is a match" in {
```

Comment:
I think this should be:
```suggestion
    "call ETMP and compare the penalty ID provided and the penalty ID in the payload - returning the oldest VAT period if there is multiple periods present" in {
```